requirements.txt

```diff
@@ -1,5 +1,5 @@
 -i https://pypi.org/simple
-alembic==1.8.1
```

Comment:
When I updated the `securesystemslib` I noticed that `tox -e requirements` failed suddenly.
I thought it was somehow connected to the newer securesytemslib versions but in their requirements files, I don't see those dependencies...

Reply:
I merged #156 
If you rebase it now, the changes only related to your PR will be clear.